server/handler/assignments.go

```diff
@@ -0,0 +1,27 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/src-d/code-annotation/server/serializer"
+)
+
+// GetAssignmentsForUserExperiment returns the experiments for the logged user and a given experiment
+// if these assignments does not already exist, they are created in advance
+func GetAssignmentsForUserExperiment() RequestProcessFunc {
```

Comment:
Instead of saying it returns something, should it be "returns a function that..."?
Also, do you mean returns the **assignments**? And "these assignments **do** not"